src/pages/index.js

```diff
@@ -96,6 +92,17 @@ const profileEditModalPopup = new PopupWithForm(
   (data) => {
     currentUserInfo.setUserInfo(data.title, data.description);
     profileEditModalPopup.close();
+    api.editCurrentUser("/users/me", data);
+  }
+);
+
+const profileAvatarEditModalPopup = new PopupWithForm(
+  "#modal_avatar-edit",
+  (data) => {
```

Comment:
Needs correction: you should also notify user about the loading process by changing the submit button text

Reply:
Done